src/content/docs/apm/agents/php-agent/php-agent-api/newrelic_set_error_group_callback.mdx

```diff
@@ -0,0 +1,238 @@
+
+The callback will provided two PHP arrays from the Agent- one containing transaction data, another containing error data.
+
+Using the provided context information, the callback must return a non-empty string based on user-defined logic that will serve as the error group name displayed in the Errors Inbox UI.
+
+<Callout variant="important">
+  - When there are multiple calls to this function in a single transaction, the PHP agent retains the callback from the last call only.
+
+  - Callbacks are registered on a per-request basis. This API should be called in a code path that is guaranteed to be executed for each request, or the callback will not be invoked.
+
+  - It is highly recommended to keep your callback function as minimal as possible. Cpu-intensive callbacks (such as database calls) or other complex logic will result in performance impacts to your application.
+</Callout>
+
+## Parameters
+
+This API accepts a single function-type callback as an argument. The provided callback must accept 2 parameters.
+
+<table>
+  <thead>
+    <tr>
+      <th width="25%">
+        API Parameter
+      </th>
+
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `$callback` (_function_)
+      </td>
+
+      <td>
+        Required. Provide a callback function that will be registered with the PHP Agent.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+<table>
+  <thead>
+    <tr>
+      <th width="25%">
+        Callback Parameters
+      </th>
+
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `$txndata` (_array_)
+      </td>
+
+      <td>
+        Required. An array of transaction data provided to your callback by the PHP Agent.
+      </td>
+    </tr>
+    <tr>
+     <td>
+       `$errdata` (_array_)
+     </td>
+     <td>
+       Required. An array of error data provided to your callback by the PHP Agent.
+     </td>
+    </tr>
+  </tbody>
+</table>
+
+### Array Key/Value pairs
+
+#### `$txndata` - PHP Agent transaction data provided to your callback
+
+<table>
+  <thead>
+    <tr>
+      <th width="25%">
+        Key
+      </th>
+      <th>
+        Value
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        "request_uri"
+      </td>
+      <td>
+        (_string_) Request URI
+      </td>
+    </tr>
+    <tr>
+      <td>
+        "path"
+      </td>
+      <td>
+        (_string_) Filepath
+      </td>
+    </tr>
+    <tr>
+      <td>
+        "method"
+      </td>
+      <td>
+        (_string_) HTTP method (`GET`, `POST`, etc.)
+      </td>
+    </tr>
+    <tr>
+      <td>
+        "status_code"
+      </td>
+      <td>
+        (_int_) HTTP status code (`200`, `404`, etc.)
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+#### `$errdata` - PHP Agent error data provided to your callback
```

Comment:
```suggestion
#### `$error_data` - PHP Agent error data provided to your callback
```

Reply:
Applied changes in bb07f23bde149d3f160cd482fb45db3236b60519